### PR TITLE
Add 0005-rfc-process document.

### PR DESCRIPTION
**Pull request by dj2**
# Summary | Résumé

Add an RFC documenting the RFC process. When approvals are required,
which approvals, how long to wait and how to announce a new RFC.

### CODE REVIEW

File: rfcs/0005-rfc-process.md
Comment (CalvinRodo) [inline rfcs/0005-rfc-process.md:31]: How many members of a community need to approve an RFC before it's ready to move to the next step.

File: rfcs/0005-rfc-process.md
Comment (fitore-cds) [inline rfcs/0005-rfc-process.md:23]: Thanks Dan, looks good! I'm wondering if we want to provide an `rfc_template.md` for an RFC, or just provide a basic guideline for consistency? E.g. sections like:
- Title
- Objective
- Motivation
- Proposal 
- ... other optional sections?

File: rfcs/0005-rfc-process.md
Comment (CalvinRodo) [inline rfcs/0005-rfc-process.md:34]: How do we determine who is **uniquely positioned** CDS is continuing to grow and it's getting more and more difficult to understand who does what and what their specific knowledge, background, and skill is.

**Review by fitore-cds — APPROVED** [review]

File: rfcs/0005-rfc-process.md
Comment (dj2) [reply rfcs/0005-rfc-process.md:23]: There is a template provided, it gives a bit of advice on what to put in each section. I'll link to it from this RFC so the things are connected. (https://github.com/cds-snc/docs/blob/main/rfc-template.md)

Comment (fitore-cds) [reply rfcs/0005-rfc-process.md:23]: Perfect, the link is great!

**Review by CalvinRodo — COMMENTED** [review]
I like this, I just think we need to be a bit more specific about the approval process, I've seen some blogs out there outlining requirements like 

 - RFC needs 2 reviewers that aren't the author before being submitted for approval
 - Instead of asking folks for approval give them veto ability or the ability to do a "Yes if.." so you can veto but you have to attach a recommendation that if adopted would result in an approval.
 
 
 https://engineering.squarespace.com/blog/2019/the-power-of-yes-if

File: rfcs/0005-rfc-process.md
Comment (dj2) [reply rfcs/0005-rfc-process.md:34]: I think this will have to come with time. There are a few people who we know have focused on areas that are important, but will also depend on people speaking up when the RFC is announced.

File: rfcs/0005-rfc-process.md
Comment (dj2) [reply rfcs/0005-rfc-process.md:31]: I like your suggestion above that 2 reviewers are required. I think a community just needs a single approver, but hopefully they've talked to their community about the RFC and aren't just deciding for themselves.

**Comment by wmoussa-cds** [comment]
Overall this is a good step: not too heavy and leaves the communities empowered.  However I believe we should just use this RFC process for the software development community and see how it goes. The other disciplines were not briefed yet on this and they need to have a say on the process. 

As for the approval, I think having the Head Of approving will give more weight. I personally blocked my calendar to review all the RFCs every week.

**Comment by dj2** [comment]
What do we do about things like #8 which puts indirect requirements on those other communities? What happens if they reject the labels we decide on?

**Comment by wmoussa-cds** [comment]
> What do we do about things like #8 which puts indirect requirements on those other communities? What happens if they reject the labels we decide on?

I've been reviewing it and was asking myself the same question. In fact, I think we have some fundamentals that still need to be sorted like: how do we work in the open, what do we share and where etc.  

Chelsea is currently driving a proposal and not every community is aligned. 

We could however go forward with developers define their own label convention with a caveat that things might change

**Comment by dj2** [comment]
That works for me. My plan was to write the accessibility RFC next, but maybe it's worthwhile to consolidate some of what the dev community has written about working in the open first.

My assumption at this point is that, all of these RFCs can, and should, change. I don't think of them as static but living documents that can be refined, retired or replaced.

**Comment by CalvinRodo** [comment]
> > What do we do about things like #8 which puts indirect requirements on those other communities? What happens if they reject the labels we decide on?
> 
> I've been reviewing it and was asking myself the same question. In fact, I think we have some fundamentals that still need to be sorted like: how do we work in the open, what do we share and where etc.

Good idea, I wrote #8 specifically to force the conversation around approval processes.

> We could however go forward with developers define their own label convention with a caveat that things might change

I agree this is probably the right approach for that specific RFC

**Review by dj2 — COMMENTED** [review]
PTAL. I updated some text, added a question about other communities and tried to put some clarification around the approval process.

**Review by wmoussa-cds — APPROVED** [review]

**Pull request merged**
